specification/service-search-api.yaml

```diff
@@ -18,4 +157,483 @@ servers:
+          type: string
+        "LastUpdatedDate":
+          type: string
+          format: "date-time"
+        "Postcode":
+          type: string
+        "Geocode":
+          $ref: "#/components/schemas/Geocode"
+        "CCG":
+          $ref: "#/components/schemas/StringArray"
+        "RelatedIAPTCCGs":
+          $ref: "#/components/schemas/StringArray"
+        "Trusts":
+          $ref: "#/components/schemas/StringArray"
+        "CCGLocalAuthority":
+          $ref: "#/components/schemas/StringArray"
+        "ServicesOffered":
+          $ref: "#/components/schemas/StringArray"
+        "ServicesProvided":
+          $ref: "#/components/schemas/StringArray"
+        "ServiceCodesProvided":
+          $ref: "#/components/schemas/StringArray"
+        "ServicesProvidedConditions":
+          $ref: "#/components/schemas/StringArray"
+        "OrgTypeConditions":
+          $ref: "#/components/schemas/StringArray"
+        "OpeningTimes":
+          $ref: "#/components/schemas/JsonEncodedString"
+        "Contacts":
+          $ref: "#/components/schemas/JsonEncodedString"
+        "VenueType":
+          $ref: "#/components/schemas/JsonEncodedString"
+        "Metrics":
+          $ref: "#/components/schemas/JsonEncodedString"
+        "GSD":
+          type: string
+        "OpeningTimesV2":
+          $ref: "#/components/schemas/OpeningTimes"
+    
+    OpeningTimes:
+      type: array
+      items:
+        type: object
+        properties:
+          "Weekday":
+            type: string
+            enum:
+              - "Monday"
+              - "Tuesday"
+              - "Wednesday"
+              - "Thursday"
+              - "Friday"
+              - "Saturday"
+              - "Sunday"
+          "Times":
+            type: string
+            example: "08:00-18:30"
+          "OffsetOpeningTime":
+            description: Minutes past midnight
+            type: number
+            example: 480
+          "OffsetClosingTime":
+            description: Minutes past midnight
+            type: number
+            example: 1110
+          "OpeningTimeType":
+            type: string
+          "AdditionalOpeningDate":
+            type: string
+          "IsOpen":
+            type: boolean
+    
+    JsonEncodedString:
+      type: string
+    
+    StringArray:
+      type: array
+      items:
+        type: string
+    
+    Geocode:
+      description: "Geocode"
+      type: object
+      properties:
+        "type":
+          type: string
+        "coordinates":
+          type: array
+          items:
+            type: number
+          minLength: 2
+          maxLength: 2
+        "crs":
+          $ref: "#/components/schemas/CoordinateReferenceSystem"
+
+    CoordinateReferenceSystem:
+      description: Co-ordinate reference system
+      type: object
+      properties:
+        "type":
+          type: string
+        "properties":
+          type: object
+          additionalProperties: true
```

Comment:
Do we know what these properties can be? If not we should find out what they are.

Reply:
We do.  It's a massively complex Azure Search query, which is documented in the description.